cdap-docs/build.sh

```diff
@@ -408,8 +408,16 @@ function build_license_dependency_pdfs() {
 }
 
 function build_standalone() {
```

Comment:
Correct. Does that make sense? Is it obvious, or do I need to add a comment or to the help?

I guess we could test for the directory and bail if not there.